File: app/src/main/java/com/tammamkhalaf/myuaeguide/categories/hereDeveloper/discoverHere/LockupEntrypoint/Images.java

package com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.LockupEntrypoint;

import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Images {

    @SerializedName("available")
    @Expose
    private Integer available;
    @SerializedName("items")
    @Expose
    private List<Item> items = null;

    public Integer getAvailable() {
        return available;
    }

    public void setAvailable(Integer available) {
        this.available = available;
    }

    public List<Item> getItems() {
        return items;
    }

    public void setItems(List<Item> items) {
        this.items = items;
    }

}

File: kmmsharedmodule/src/commonMain/kotlin/com/tammamkhalaf/kmmsharedmodule/categories/hereDeveloper/discoverHere/LockupEntrypoint/Images.kt
package com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.LockupEntrypoint

import com.google.gson.annotations.Expose

class Images {
    @SerializedName("available")
    @Expose
    private var available: Int? = null

    @SerializedName("items")
    @Expose
    private var items: List<com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.LockupEntrypoint.Item>? = null
    fun getAvailable(): Int? {
        return available
    }

    fun setAvailable(available: Int?) {
        this.available = available
    }

    fun getItems(): List<com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.LockupEntrypoint.Item>? {
        return items
    }

    fun setItems(items: List<com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.LockupEntrypoint.Item>?) {
        this.items = items
    }
}